src/pages/AdvancedOrders/index.tsx

```diff
@@ -29,12 +35,20 @@ export default function AdvancedOrdersPage() {
     return <Navigate to={parameterizeTradeRoute(tradeContext, RoutesEnum.SWAP)} />
   }
 
+  const disablePriceImpact =
+    primaryFormValidation === TradeFormValidation.QuoteErrors ||
```

Comment:
Nitpick:
The conditions with `primaryFormValidation` are duplicated in Limit/Advanced.
We could move it at least in a const:
```
const PRICE_IMPACT_IGNORE_CASES = [TradeFormValidation.QuoteErrors, TradeFormValidation.CurrencyNotSupported, TradeFormValidation.WrapUnwrapFlow]
```

Reply:
That's true, I forgot to refactor it. Thanks for the reminder